packages/i18n/src/index.js

```diff
@@ -35,7 +35,7 @@ export function setLocaleData( localeData = { '': {} }, domain = 'default' ) {
 		} );
 	}
 
-	i18n.options.locale_data[ domain ] = has( i18n.options.locale_data, domain )
+	i18n.options.locale_data[ domain ] = domain in i18n.options.locale_data
```

Comment:
so @aduth I ended up using `in` because it appears that's the [most performant option](https://jsperf.com/lodash-has-vs-hasownproperty-vs-in-object)

Reply:
Well to be more accurate, its the most performant option in firefox and chrome.  In Safari `hasOwnProperty` wins.   I'm happy to use whatever you prefer.

Reply:
It's important to understand that `in` has different behavior than `hasOwnProperty` though, where the former traverses up through the prototype chain of the object.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

As implemented here, this is susceptible to errors / oddities when doing something like assigning `valueOf` as a domain, or any other member of the [object prototype](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/prototype).

```js
'valueOf' in {}; // true
( {} ).hasOwnProperty( 'valueOf' ); // false
```

This line of code is likely to be called only a handful of times for an entire page session, so optimizing for performance here is a bit overkill.

Reply:
Related: https://github.com/WordPress/packages/pull/85

Reply:
gotcha, makes sense, changed it.